client/src/components/manageProjects/editProject.js

```diff
@@ -92,7 +92,7 @@ const EditProject = ({
         />
       </div>
       <ProjectForm
-        arr={simpleInputs}
+        arr={[...simpleInputs, ...additionalInputsForEdit]}
```

Comment:
In order for state to be passed to the form the info needs to be destructed off `projectToEdit`

```js
const [formData, setFormData] = useState({
    name: projectToEdit.name,
    description: projectToEdit.description,
    location: projectToEdit.location,
    githubIdentifier: projectToEdit.githubIdentifier,
    githubUrl: projectToEdit.githubUrl,
    slackUrl: projectToEdit.slackUrl,
    googleDriveUrl: projectToEdit.googleDriveUrl,
    hflaWebsiteUrl: projectToEdit.hflaWebsiteUrl,
    // ------ Need these fields to be destructured
    partners: projectToEdit.partners,
    managedByUsers: projectToEdit.managedByUsers,
    projectStatus: projectToEdit.projectStatus,
    googleDriveId: projectToEdit.googleDriveId,
    createdDate: projectToEdit.createdDate,
    // ------ 
  });